.github/workflows/build.yml

```diff
@@ -609,9 +609,7 @@ jobs:
       #     -d@test/tmp/testResults.json || echo "Curl command failed but continuing"
 
   typescript-tracing-tests:
-    if: >
-      (github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository) ||
-      (github.event_name == 'push' && github.ref == 'refs/heads/master')
+    if: ${{ github.event_name == 'pull_request' || (github.event_name == 'push' && github.ref == 'refs/heads/master') }}
```

Comment:
`github.event.pull_request.head.repo.full_name == github.repository` was included to ensure the action doesn't run on other repository (specially when you have a fork). I don't fully remember the reason but I think it is because it is dependant on some external element not available on other forks

Reply:
Reverted this change. While possible, it is tricky to use a branch from a fork repo as rust dependency.

In my opinion it is fine to skip this job on fork pull requests